packages/wdio-utils/src/shim.ts

```diff
@@ -4,9 +4,14 @@ const log = logger('@wdio/utils:shim')
 
 let inCommandHook = false
 let hasWdioSyncSupport = false
-let runSync = null
+let runSync: (this: unknown, fn: Function, repeatTest: any, args: unknown[]) => (resolve: Function, reject: Function) => unknown
```

Comment:
This now changed from null to undefined, is that what we want?

Reply:
yes, there is no different in the behavior.